google/cloud/workflows/quickstart/README.md

```diff
@@ -5,21 +5,19 @@ client library in your own project. These instructions assume that you have
 some experience as a C++ developer and that you have a working C++ toolchain
 (compiler, linker, etc.) installed on your platform.
 
-* Packaging maintainers or developers who prefer to install the library in a
+- Packaging maintainers or developers who prefer to install the library in a
```

Comment:
Should we fix the generator to more closely follow this standard?

Reply:
Yes, eventually, I think.  See above.